backend/controllers/followups.js

```diff
@@ -4,7 +4,10 @@ const Followup = require("mongoose").model("Followup")
 const situation = require("./answers")
 const pollResult = require("../lib/mattermost-bot/poll-result")
 
-const excludeFields = ["surveys.accessToken"].join(" -").replace(/^/, "-")
+// TODO next line is to be updated once tokens are used globally
+const excludeFields = ["accessToken", "surveys.accessToken"]
+  .join(" -")
+  .replace(/^/, "-")
```

Comment:
Le code ci-dessus ne me parait pas utile 
pourquoi ne pas directement faire  : 
```suggestion
const excludeFields = "-accessToken -surveys.accessToken"
```